GDJS/Runtime/gd.js

```diff
@@ -358,6 +358,11 @@ gdjs.objectsListsToArray = function(objectsLists) {
   return result;
 };
 
+/**
+ * Is true on previews.
+ */
+gdjs.isPreview = false;
```

Comment:
Avoid global variables as much as possible, I know some game engines that are overbloated of stuff in a global namespace. Prefer to put it into `RuntimeGame`.

It's of course not the case, but it might happen that in the future we could run more than one RuntimeGame at the same time :)